Description: remove example from Cargo.toml
 The directory examples/ is not shipped by upstream
 with the crate
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-06-11

--- rust-mockall-0.12.1.orig/Cargo.toml
+++ rust-mockall-0.12.1/Cargo.toml
@@ -55,11 +55,6 @@ file = "../CHANGELOG.md"
 replace = "{{date}}"
 search = "ReleaseDate"
 
-[[example]]
-name = "serde"
-crate-type = ["lib"]
-path = "examples/serde.rs"
-
 [dependencies.cfg-if]
 version = "1.0"
 
